src/components/Pages/PeoplePage.tsx

```diff
@@ -0,0 +1,69 @@
+import { useSearchParams } from 'react-router-dom';
+import { useMemo } from 'react';
+import { PeopleFilters } from '../PeopleFilters';
+import { Loader } from '../Loader';
+import { PeopleTable } from '../PeopleTable';
+import { usePeople } from '../../store/peopleContext';
+
+export const PeoplePage = () => {
+  const {
+    people, loading, loadingError, filterPeople,
+  } = usePeople();
+  const [searchParams] = useSearchParams();
+  const query = searchParams.get('query') || '';
+  const sex = searchParams.get('sex') || '';
+  const centuries = searchParams.getAll('centuries') || [];
+  const sort = searchParams.get('sort') || '';
+  const order = searchParams.get('order') || '';
+  const filteredPeople = useMemo(() => {
+    return filterPeople({
+      query, sex, centuries, sort, order,
+    });
+  }, [people, query, centuries, sex, sort, order]);
```

Comment:
let's provide some spaces for making code more readable 
```suggestion
  const {
    people, loading, loadingError, filterPeople,
  } = usePeople();
  
  const [searchParams] = useSearchParams();
  const query = searchParams.get('query') || '';
  const sex = searchParams.get('sex') || '';
  const centuries = searchParams.getAll('centuries') || [];
  const sort = searchParams.get('sort') || '';
  const order = searchParams.get('order') || '';
  
  const filteredPeople = useMemo(() => {
    return filterPeople({
      query, sex, centuries, sort, order,
    });
  }, [people, query, centuries, sex, sort, order]);
```